src/styles.scss

```diff
@@ -51,10 +51,6 @@ h2 {
         width: 130px;
       }
 
-      > div:last-of-type {
```

Comment:
I'm not sure what was the reason to delete this, but it breaks timestamp tooltips
![screenshot 2018-10-15 at 11 53 50](https://user-images.githubusercontent.com/1536298/46943929-1232c880-d071-11e8-9607-f757dd69d7d0.png)

Reply:
I deleted that part because it caused a problem with horizontal scrolling on small screens, specifically when showing the hashes of the blocks.
![space](https://user-images.githubusercontent.com/34079003/46952125-fff86080-d057-11e8-9a56-17b77463b091.png)

Thanks for noticing the problem. I used another solution in the last commit.